projects/js-packages/licensing/components/activation-screen/index.jsx

```diff
@@ -77,6 +78,16 @@ const ActivationScreen = props => {
 		}
 	}, [ availableLicenses ] );
 
+	const [ activePlugins, setActivePlugins ] = useState( [] );
+
+	useEffect( () => {
+		apiFetch( { path: 'wp/v2/plugins' } ).then( plugins =>
+			setActivePlugins(
+				plugins.filter( plugin => plugin.status === 'active' ).map( plugin => plugin.name )
+			)
+		);
+	}, [] );
```

Comment:
The tests are broken here because `apiFetch` is unavailable. I thought maybe it was a dependency thing, but that hasn't solved it, and thinking about it some more, we don't want to call this in the tests anyway. 

I wonder if it would be worth making this a custom hook, like `useActivePlugins`, and then we could use that in the ProductDetails component, rather than passing it down as a prop. It would also mean we could mock it for the tests.

I think @gmjuhasz has done a bit more of this sort of thing, so maybe he could help out here?